proto/kvrpcpb.proto

```diff
@@ -859,6 +901,9 @@ message LockInfo {
     bool use_async_commit = 8;
     uint64 min_commit_ts = 9;
     repeated bytes secondaries = 10;
+    // Hint the client that resolving lock is not needed for this lock. For `PessimisticLock`
+    // requests only.
+    bool skip_resolving_lock = 11;
```

Comment:
Could it be put in `PessimisticLockKeyResult`? I think it's better not to pollute `LockInfo` because of one request.